### PR TITLE
[M] Corrected a few issues in the Ansible playbook used by Vagrant

### PR DESCRIPTION
**Pull request by Ceiu**
- Removed extraneous environment and Ansible variable declarations
- Corrected syntax on the custom role task

### CODE REVIEW

**Comment by Ceiu** [comment]
This PR is reliant on changes made to the Ansible role here: https://github.com/candlepin/ansible-role-candlepin/pull/23

**Review by nikosmoum — APPROVED** [review]
ack: will merge once the other PR is merged first

**Comment by nikosmoum** [comment]
Merging this since the only failure is sonarqube infrastructure related, and is unrelated/unaffected to this

**Pull request merged**
